src/util/git/git-spawn.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import * as child from '../child.js';
+
+// Suppress any password prompts since we run these in the background
+const env = {
+  GIT_ASKPASS: '',
+  GIT_TERMINAL_PROMPT: 0,
+  GIT_SSH_COMMAND: 'ssh -oBatchMode=yes',
+  ...process.env,
+};
+
+export const spawn = (args: Array<string>, opts?: child_process$spawnOpts = {}): Promise<string> => {
```

Comment:
Alright. Is it too hard to mock this without putting it into a separate module? If so I'm okay keeping this, it just looked a bit off to me, that's all.